src/commands.ts

```diff
@@ -1286,6 +1286,16 @@ export const ignoreMetaNodeBackspaceCommand = (
   )
 }
 
+export const ignoreNodeCommands = (state: ManuscriptEditorState) => {
```

Comment:
Why is this needed? Also, it says `ignoreNodeCommands` but from the implementation, I see it ignores `section_title` nodes.

Reply:
ohh updated to `bibliography_section`